src/ScaleformMCM.cpp

```diff
@@ -155,9 +155,14 @@ namespace ScaleformMCM {
 			
 			const char* formIdentifier	= args->args[0].GetString();
 			const char*	propertyName	= args->args[1].GetString();
+			const char* scriptName		= nullptr;
+			if (args->numArgs == 3 && (args->args[2].GetType() == GFxValue::kType_String))

```

Comment:
Let's make `scriptName` a required parameter for this function so that we can simplify the logic here.